src/verify_cert.rs

```diff
@@ -58,9 +59,10 @@ impl<'a> ChainOptions<'a> {
         &self,
         end_entity: &'a EndEntityCert<'a>,
         time: UnixTime,
+        verify_path: Option<&dyn Fn(&VerifiedPath<'_>) -> Result<(), Error>>,
```

Comment:
I think it might be worth adding a little bit more detail about the `verify_path` argument in the rustdoc comment above.

In particular maybe we should note that it can never accept a path that would otherwise have been rejected for some reason, it can only ever discard a validated path that would have been used. It also can't halt path building entirely. When it returns an err we discard the validated path but do continue searching.

Reply:
Update looks good, thank you